data-reconciliation-app/docs/adr/05-Reporting.md

```diff
@@ -92,23 +89,23 @@ getDataByKey(userId: string, key: string) : SummaryResult   // for a single reco
 
 ### Summary Result Object
 
-The SummaryResult Object will be the final output object in the report.
+The SummaryResult object will be the final output in the report. In the report, the key and value will match the schema defined.
```

Comment:
maybe add a link to the code where this schema is defined?

Reply:
link added